### DISCUSSION TITLE
Usage of defaultgl field on suppliers table

### DISCUSSION THREAD

**Discussion by pakricard** [timschofield/webERP#279] [Development Discussion & Specification]
Hi all:

Regarding the field defaultgl on the suppliers table:

It was added by @timschofield on 19/06/2018 on commit 01898ee, and it refers to http://www.weberp.org/forum/showthread.php?tid=8169&pid=14931#pid14931 but now that the forum is gone there is no context.

Commit message reads: "Ensure that when entering a GL suppliers invoice the default account is pre-selected"

apparently it is only used in line 103 of SuppTransGLAnalysis.php

- What is its practical use? 
- Should we have a "None" by default? 
- What am I missing here?

**Comment by dalers** [comment]
I might have found the post ("clone invoice") in the internet wayback machine https://web.archive.org/web/20201025015925/http://www.weberp.org/forum/showthread.php?tid=8169

Does this help?

**Comment by pakricard** [comment]
A brief summary of that post, titled "Clone invoice"

- 18/06/2018: A few years ago a customer asked for a default GL code by supplier. So for instance if you went to create an invoice for your electricity supplier it defaulted to the electricity GL account and all you had to do was fill in the date reference and the amount. 

So, IMO, is OK to keep it, but the suppliers scripts should allow defaultgl to be empty (both in supplier maintenance and any supplier related script

Moved it to issues to be solved
